examples/ASDKgram/Sample/AppDelegate.h

```diff
@@ -0,0 +1,21 @@
+//
+//  AppDelegate.h
+//  ASDKgram
+//
+//  Created by Hannah Troisi on 2/16/16.
+//  Copyright © 2016 Hannah Troisi. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@protocol PhotoFeedViewControllerProtocol <NSObject>
+- (void)resetAllData;
+@end
+
+@interface AppDelegate : UIResponder <UIApplicationDelegate>
+
+@property (strong, nonatomic) UIWindow *window;
```

Comment:
I think we do not need to expose either of these properties, and they can be internal only. In fact, they probably don't need properties and can be just instance variables.